terraform/examples/README.md

```diff
@@ -0,0 +1,10 @@
+# Terraform Examples
+
+This directory contains examples of how to use Terraform to deploy Firezone
+infrastructure.
+
+## Examples
+
+- [gcp_nat_gateway](./gcp_nat_gateway): This example shows how to deploy one or
+  more Gateways in a single GCP VPC that are configured to egress traffic
```

Comment:
```suggestion
  more Firezone Gateways in a single GCP VPC that are configured to egress traffic
```
Only suggesting this change because the term 'gateway' is slightly overloaded here.